backend/hitas/tests/apis/test_api_thirty_year_regulation.py

```diff
@@ -511,6 +511,13 @@ def test__api__regulation__automatically_release__partial(api_client: HitasAPICl
         apartment__building__real_estate__housing_company__state=HousingCompanyState.LESS_THAN_30_YEARS,
     )
 
+    # For some reason, financing method old ruleset flag may not always get set with the factory,
+    # resulting in the test being flaky, so try to fix the flakiness with explicit setting
+    sale_1.apartment.housing_company.financing_method.old_hitas_ruleset = False
+    sale_1.apartment.housing_company.financing_method.save()
+    sale_2.apartment.housing_company.financing_method.old_hitas_ruleset = True
+    sale_2.apartment.housing_company.financing_method.save()
+
```

Comment:
Fix and explanation for this test is already included in 431e5a6e48722fa797aa49ce687c492dd22e7655 :)

Reply:
Yeah, dropped the commit.